docs/api/api-handler/api-handler.md

```diff
@@ -0,0 +1,46 @@
+---
+id: api-handler
+title: API Handler
+description: This article describes the structure of an API handler function and its arguments.
+---
+
+An API handler function has the following structure:
+
+```js
+export default async (req, res) => {
+  // ...
+}
+```
+
+The handler receives a request and response objects. See the sections below for the information on the API exposed through these objects.
+
+### Request
+
+The request object exposes the following fields:
+
+| Field     | Description                                                                              |
+| --------- | ---------------------------------------------------------------------------------------- |
+| `resolve` | The [reSolve context](resolve-context.md) object that contains reSolve API and metadata. |
+| `method`  | The request's HTTP method.                                                               |
+| `path`    | The request URL path.                                                                    |
+| `body`    | The request body.                                                                        |
+| `cookies` | An object that contains the attached cookies as key-value pairs.                         |
+| `headers` | An object that contains the request's HTTP headers as key-value pairs.                   |
+| `query`   | An object that contains the request's query string parameters as key-value pairs.        |
+
+### Response
+
+The request object exposes the following functions:
+
+| Function                               | Description                                 |
+| -------------------------------------- | ------------------------------------------- |
+| `status(code)`                         | Specifiers the response status code.        |
+| `getHeader(key)`                       | Get a response header by key.               |
+| `setHeader(key, value)`                | Sets a response header.                     |
+| `text([content] [, encoding])`         | Specifies content for a text-type response  |
+| `json([content])`                      | Specifies content for a JSON-type response. |
+| `end([content] [, encoding])`          | Ends the response process.                  |
+| `file(content, filename [, encoding])` | Specifies a file to write to response.      |
```

Comment:
1. Please add an article ("the") before "response".

2. For my understanding, can you write a file to a response? Just checking.

Reply:
I rewrote this sentence  as follows:
>Specifies a file to send as a response.

Reply:
> Specifies a file to send as a response.

Sounds good. Thank you!